src/components/ModalWithForm/LoginModal/LoginModal.jsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from "react";
+import "./LoginModal.css";
+import ModalWithForm from "../ModalWithForm";
+
+const LoginModal = ({
+  handleLogin,
+  isOpen,
+  closeActiveModal,
+  handleSignup,
+  handleOutsideClick,
+}) => {
+  const [data, setData] = useState({
+    email: "",
+    password: "",
+  });
+
+  const [errorMessage, setErrorMessage] = useState(""); // For error handling
```

Comment:
* Please, add **validation** for the forms https://snipboard.io/elPgxq.jpg . The criteria say: Until there is valid data in all the input fields on the "Sign up" and "Sign in" pages, the form submit buttons remain disabled.